docs/discovery.md

```diff
@@ -212,7 +212,7 @@ You can contribute a new discovery backend to Swarm. For information on how to d
 
 ## Docker Swarm documentation index
 
-- [User guide](index.md)
-- [Sheduler strategies](/scheduler/strategy.md)
-- [Sheduler filters](/scheduler/filter.md)
-- [Swarm API](/api/swarm-api.md)
+- [User guide](https://docs.docker.com/swarm/)
```

Comment:
Hi @peggyl everything in the `docs` directory is intended to work on `docs.docker.com` -- GitHub repos serve individual pages.  They aren't intended to work like a web. GitHub itself does this: https://goo.gl/Uwh9q3

So, if you just want to do the one typo, please do and thank you for your PR.